File: app/src/main/java/view/viewdemo/activity/BaseActivity.java
package view.viewdemo.activity;

import android.os.Bundle;

import android.util.Log;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

/**
 * Created by zyand on 2019/4/20.
 */

public class BaseActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Log.d("Activity:",getClass().getSimpleName());
    }
}

File: app/src/main/java/com/example/kotlindemo04/ui/BaseActivity.kt
package com.example.kotlindemo04.ui

import android.os.Bundle
import android.os.PersistableBundle
import android.util.Log
import androidx.activity.ComponentActivity
import androidx.appcompat.app.AppCompatActivity

open class BaseActivity: AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?, persistentState: PersistableBundle?) {
        super.onCreate(savedInstanceState, persistentState)
        Log.d("zlc","当前的页面是:${javaClass.simpleName}")
    }
}